<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKD_Qt5_InterfaceClass</class>
 <widget class="QMainWindow" name="QKD_Qt5_InterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QKD_Qt5_Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="CurveStat" name="graphicsView"/>
          <widget class="CurveStat" name="graphicsView_2"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="CurveStat" name="graphicsView_3"/>
          <widget class="CurveStat" name="graphicsView_4"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
   </widget>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowVertices"/>
   <addaction name="actionNewRand"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShowVertices">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ShowVertices</string>
   </property>
  </action>
  <action name="actionNewRand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NewRand</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CurveStat</class>
   <extends>QGraphicsView</extends>
   <header>CurveStat.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QKD_Qt5_Interface.qrc"/>
 </resources>
 <connections/>
</ui>
